tasks/request_files/schemas/output.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
We may want to revisit this later after we do the archiving workflow functionality to see if we can get away with a close to common output for both the archive and recovery path from the step function. I would maybe write a card on the backlog for investigation to do that analysis for future work.

Reply:
Created, though I'm not entirely sure I'm parsing your request correctly. https://bugs.earthdata.nasa.gov/browse/ORCA-200